wire-runtime/src/test/java/com/squareup/wire/protos/redacted/NotRedacted.java

```diff
@@ -50,13 +90,22 @@ public boolean equals(Object other) {
   public int hashCode() {
     int result = hashCode;
     if (result == 0) {
-      result = a != null ? a.hashCode() : 0;
+      result = result * 37 + (a != null ? a.hashCode() : 0);
```

Comment:
This might be the microist of mico optimizations and I doubt any modern CPU will actually issue a multiply operation on a zero operand but for the sake of the diff I'll revert.